runtime/src/epoch_stakes.rs

```diff
@@ -124,8 +124,9 @@ impl EpochStakes {
     }
 }
 
+#[cfg_attr(feature = "frozen-abi", derive(AbiExample, AbiEnumVisitor))]
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub(crate) enum VersionedEpochStakes {
+pub enum VersionedEpochStakes {
```

Comment:
Is this change in `pub`licness necessary? I don't see anything in this diff outside `solana-runtime`

Reply:
yup

```
warning: type `epoch_stakes::VersionedEpochStakes` is more private than the item `bank::BankFieldsToSerialize::versioned_epoch_stakes`
   --> runtime/src/bank.rs:507:5
    |
507 |     pub versioned_epoch_stakes: HashMap<u64, VersionedEpochStakes>,
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ field `bank::BankFieldsToSerialize::versioned_epoch_stakes` is reachable at visibility `pub`
    |
note: but type `epoch_stakes::VersionedEpochStakes` is only usable at visibility `pub(crate)`
   --> runtime/src/epoch_stakes.rs:129:1
    |
129 | pub(crate) enum VersionedEpochStakes {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    = note: `#[warn(private_interfaces)]` on by default

warning: type `epoch_stakes::VersionedEpochStakes` is more private than the item `serde_snapshot::ExtraFieldsToSerialize::versioned_epoch_stakes`
   --> runtime/src/serde_snapshot.rs:417:5
    |
417 |     pub versioned_epoch_stakes: HashMap<u64, VersionedEpochStakes>,
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ field `serde_snapshot::ExtraFieldsToSerialize::versioned_epoch_stakes` is reachable at visibility `pub`
    |
note: but type `epoch_stakes::VersionedEpochStakes` is only usable at visibility `pub(crate)`
   --> runtime/src/epoch_stakes.rs:129:1
    |
129 | pub(crate) enum VersionedEpochStakes {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Ah, thanks